Makefile

```diff
@@ -12,7 +12,6 @@ endif
 test: clean
 	./manage.py test --settings=$(test_settings) --with-coverage --with-ignore-docstrings \
 		--exclude-dir=notesserver/settings --cover-inclusive --cover-branches \
-		--ignore-files=search_indexes.py --ignore-files=highlight.py\
```

Comment:
@symbolist this is not fixed in latest version. I will comment the line and add an explanation.